spec/controllers/selfservice/advisers_controller/create_spec.rb

```diff
@@ -23,7 +23,7 @@ module SelfService
         end
 
         it 'adds a success flash message' do
-          expect(flash[:notice]).to eq('Saved successfully')
+          expect(flash[:notice]).to eq(I18n.t('self_service.adviser_edit.saved'))
```

Comment:
And I should add, is that ok? It seemed like that was "the way things are done".